### PR TITLE
[HOPS-1563] and [HOPS-1562]

### PR DESCRIPTION
**Pull request by maismail**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added and passed (for bug fixes / features)
- [x] HOPS JIRA issue has been opened for this PR
- [x] All commits have been squashed down to a single commit
- [x] The commit message has the following format: [HOPS-XXX] message

* **Post a link to the associated JIRA issue**
https://hopshadoop.atlassian.net/browse/HOPS-1562
https://hopshadoop.atlassian.net/browse/HOPS-1563

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fixes

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by berthoug** [comment]
Jenkins test this please.

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirAclOp.java [hunk 1]
```diff
@@ -70,14 +70,6 @@ public Object performTask() throws IOException {
...
         AclStorage.validateAclSpec(aclSpec);
         INode inode = FSDirectory.resolveLastINode(iip);
-        if (aclSpec.size() == 1 && aclSpec.get(0).getType().equals(AclEntryType.MASK)) {
-          //HOPS: We allow setting
-          FsPermission fsPermission = inode.getFsPermission();
-          inode.setPermission(new FsPermission(fsPermission.getUserAction(), aclSpec.get(0).getPermission(),
-              fsPermission
-                  .getOtherAction()));
-          return AclStorage.readINodeLogicalAcl(inode);
-        }
```
Comment (berthoug) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirAclOp.java:80]: Do you know why we were doing this?

Comment (maismail) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirAclOp.java:80]: No idea why this code was there, let's see after testing if it affected some of the test cases. I removed it for now to be consistent with HDFS Acls

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

**Pull request merged**